eng/Versions.props

```diff
@@ -6,6 +6,7 @@
     <MajorVersion>8</MajorVersion>
     <MinorVersion>1</MinorVersion>
     <PatchVersion>1</PatchVersion>
+    <PackageValidationBaselineVersion>$(MajorVersion).$(MinorVersion).$([MSBuild]::Subtract($(PatchVersion), 1))</PackageValidationBaselineVersion>
     <WcfAssemblyVersion>8.1.1.0</WcfAssemblyVersion>
```

Comment:
The next version will be 8.1.2. It looks like @HongGit doesn't update this version number until we're ready to build, so the comparison should be with 8.1.1 and not 8.1.0.